src/sentry/integrations/github/utils.py

```diff
@@ -39,7 +43,7 @@ def get_next_link(response: Response) -> str | None:
 
     for link in links:
         # If there is a 'next' link return the URL between the angle brackets, or None
-        if 'rel="next"' in link:
+        if f'rel="{link_text}"' in link:
```

Comment:
The changes in here are not necessary since the purpose at some point was to pull the last page. It might be useful in the future.